__tests__/reducers-test.ts

```diff
@@ -2,30 +2,104 @@ import counterReducer from '../src/reducers';
 import {
   addCounter,
   decreaseCount,
+  getCounters,
```

Comment:
I believe this should be named a bit differently. "get" doesn't fully describe that this is actually loading from an API. Maybe "download"?